spk/cops/src/service-setup.sh

```diff
@@ -5,19 +5,15 @@ PACKAGE_NAME="com.synocommunity.packages.${SYNOPKG_PKGNAME}"
 
 # Others
 CFG_FILE_NAME="config_local.php"
+SECURITY_SETTINGS_FILE_NAME=".htaccess"
 DEFAULT_CFG_FILE="${SYNOPKG_PKGDEST}/${CFG_FILE_NAME}.synology"
-DSM6_WEB_DIR="/var/services/web"
-if [ "${SYNOPKG_DSM_VERSION_MAJOR}" -ge 7 ]; then
-   WEB_DIR="/var/services/web_packages"
-else
-   WEB_DIR="${DSM6_WEB_DIR}"
-fi
+
+WEB_DIR="/var/services/web_packages"
+SECURITY_SETTINGS_FILE="${WEB_DIR}/${SYNOPKG_PKGNAME}/${SECURITY_SETTINGS_FILE_NAME}"
 CFG_FILE="${WEB_DIR}/${SYNOPKG_PKGNAME}/${CFG_FILE_NAME}"
-BUILDNUMBER="$(/bin/get_key_value /etc.defaults/VERSION buildnumber)"
 
 USER="http"
 GROUP="http"
-PHP_CONFIG_LOCATION="$([ "${BUILDNUMBER}" -ge "7135" ] && echo -n /usr/local/etc/php56/conf.d || echo -n /etc/php/conf.d)"
 
 service_preinst ()
```

Comment:
@smaarn to show the error message to the user, `service_preinst` must be replaced by `validate_preinst`.

otherwise the installation does not terminate and the error message is shown in the intallation logs only.

So this package requires that the shared folder is created before the installation of cops and the calibre data must already exist in the shared folder.
This is worth to mention in the install wizard too (not only the need of trailing `/`).

A more enhanced wizard would test the existance of the folder and metadata.db in a wizard validation function of the first page....

Reply:
Had a quick look at the Synology APIs but couldn't find any function which would allow such a thing. We'll be doing that async.

In the meantime I fixed the invalid override.

Reply:
> Had a quick look at the Synology APIs but couldn't find any function which would allow such a thing. We'll be doing that async.
> 

We ca do this later.

BTW:
wizard is only documented in the DSM6 developer guide

a validation function is documented as
```json
{
  "step_title": "Step2",
  "items": [{
    "type": "multiselect",
    "desc": "a check group",
    "subitems": [{
      "key": "check1",
      "desc": "Check button 1"
    }, {
      "key": "check2",
      "desc": "Check button 2",
      "defaultVaule": true,
      "validator": {
        "fn": "{var v=arguments[0]; if (!v) return 'Check this';return true;}"
      }
    }]
  },
```